cmake/compiler/gcc/target.cmake

```
@@ -34,7 +34,21 @@ if(NOT DEFINED NOSYSDEF_CFLAG)
   set(NOSYSDEF_CFLAG -undef)
 endif()
 
-foreach(file_name include/stddef.h include-fixed/limits.h)
+# GCC-13, does not install limits.h on include-fixed anymore
+# https://gcc.gnu.org/git/gitweb.cgi?p=gcc.git;h=be9dd80f933480
+# Add check for GCC version >= 13.1
+execute_process(
+    COMMAND ${CMAKE_C_COMPILER} -dumpversion
+    OUTPUT_VARIABLE TEMP_COMPILER_VERSION
+    )
+
+if(${TEMP_COMPILER_VERSION} VERSION_GREATER_EQUAL 13.1.0)
```

Please quote to ensure that you don't get a spurious error that `if()` is called with invalid args in case the out variable is empty.
```suggestion
if("${temp_compiler_version}" VERSION_GREATER_EQUAL 13.1.0)
```

---

Done.